lib/DDG/Spice/Dockerhub.pm

```diff
@@ -0,0 +1,30 @@
+package DDG::Spice::Dockerhub;
+# ABSTRACT: Search for Docker Hub images
+
+use DDG::Spice;
+use strict;
+use warnings;
+
+spice is_cached => 1;
+spice proxy_cache_valid => '200 1d';
+spice wrap_jsonp_callback => 1;
+spice to => 'https://index.docker.io/v1/search?q=$1';
+
+triggers startend => [
```

Comment:
When I was looking at the unanswered query data, I found that `docker <image_name>` came up quite a bit.